internal/query/execution.go

```diff
@@ -386,6 +390,7 @@ func scanExecutionTargets(rows *sql.Rows) ([]*ExecutionTarget, error) {
 			endpoint         = &sql.NullString{}
 			timeout          = &sql.NullInt64{}
 			interruptOnError = &sql.NullBool{}
+			signingKey       = &sql.NullString{}
```

Comment:
this way JSONB just get a string, and the whole object is used as signing key

Reply:
Changed that the signing key is decrypted after the query